message.go

```diff
@@ -142,9 +142,9 @@ func (m *Message) GetData() []byte {
 }
 
 // GetLinkName returns associated link name or empty string if receiver or link is not defined.
-func (m *Message) GetLinkName() string {
-	if m.receiver != nil && m.receiver.link != nil {
-		return m.receiver.link.key.name
+func (m *Message) LinkName() string {
```

Comment:
Will release this with the upcoming breaking change that removes `HandleMessage()`